package.json

```diff
@@ -34,7 +34,6 @@
   "devDependencies": {
     "@babel/core": "7.12.3",
```

Comment:
Do we need `babel` dependencies any more (`@babel/core grunt-babel`)? I'm assuming that editor.js was the only file using ES6/babel-powered JS syntax - is that correct? Looks like babel is (mostly) removed from Gruntfile (in this PR). 

There are two options:

1. Keep babel installed and run the task, even if it's not needed. This is "future proof", we have babel running and can update existing code to use ES6 when fixing bugs.
2. Completely remove babel entirely, since it's not strictly needed right now. 

@Aljullu Up to you which approach we take here. From a quick look at the gruntfile, I think we need to remove `grunt.loadNpmTasks( 'grunt-babel' );` as well (if you decide on option 2).

Reply:
Good point! Removing them sounds like the best option to me, it makes the codebase simpler and decreases the maintenance burden. I have removed babel completely in https://github.com/woocommerce/storefront/pull/1525/commits/9dec992aeba91ec85127e2a0662810d245f678cf, we can always install it back if we need it in the future.